code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs

```diff
@@ -203,7 +203,7 @@ mkIMField _ _ l _ = error $ "Label " ++ show l ++ " not supported " ++
 firstPair' :: InclUnits -> DataDefinition -> ListTuple
 firstPair' IgnoreUnits d  = (P $ eqSymb $ d ^. defLhs, Flat $ phrase d, Nothing)
 firstPair' IncludeUnits d =
-  (P $ eqSymb $ d ^. defLhs, Flat $ phrase d +:+ sParen (toSentenceUnitless $ d ^. defLhs), Nothing)
+  (P $ eqSymb $ d ^. defLhs, Flat $ phrase (d ^. defLhs) +:+ sParen (toSentenceUnitless $ d ^. defLhs), Nothing)
```

Comment:
This fixes the *symptom* but not the bug! It's nice that the fix is fairly generic though. So I'll accept this as a proper fix for the purposes of this PR, but it is not really a fix in that the 'phrase' in the given Chunk should never have been "Chasles' theorem", which points to bad re-use.